Documentation/admin/install-multi.md

```diff
@@ -0,0 +1,195 @@
+  labels:
+    stage: production
+    cloud: aws
+    region: us-west-1
+    role: replica
+  annotations:
+    multicluster.coreos.com/console-url: https://west-coast-production.west.example.com
+    multicluster.coreos.com/directory: true
+spec:
+  KubernetesAPIEndpoints:
+    ServerEndpoints:
+      - ServerAddress: https://west-coast-production-api.west.example.com:443
+```
+
+Last, save the file and submit it to the cluster:
+
+```
+kubectl apply -f clusters/west-coast-production.yaml
+cluster “west-coast-production” created
+```
+
+Before setting up a replica cluster, register it in the directory. Change the labels and endpoints as necessary. Be sure that  the `multicluster.coreos.com/directory` annotation is only included on the directory cluster. Submit it to the cluster:
+
+```
+kubectl apply -f clusters/east-coast-production.yaml
+cluster “east-coast-production” created
+```
+
+List the clusters:
+
+```
+kubectl get clusters
+NAME                    AGE
+west-coast-production   1m
+east-coast-production   10s
+```
+
+The directory cluster is all set up!
+
+## Generate credentials
+
+Next, create credentials for replica clusters to read the directory objects. To do this, use a service account created on the directory cluster, then export it for use on the replica cluster.
+
+Create a Service Account for the replica cluster. By convention, give the Service Account the same name as the replica cluster.
+
+```
+kubectl create serviceaccount -n tectonic-multi-cluster east-coast-production
+serviceaccount "east-coast-production" created
+
+kubectl get serviceaccounts -n tectonic-multi-cluster
+NAME                    SECRETS   AGE
+default                 1         15m
+east-coast-production   1         12m
+```
+
+Bind the new Service Account to the “view clusters” role:
+
+```
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: east-coast-production-view-clusters
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: tectonic-view-clusters
+subjects:
+- kind: ServiceAccount
+  name: east-coast-production
+  namespace: tectonic-multi-cluster
+
+kubectl apply -f east-coast-production-binding.yaml
+Clusterrolebinding "east-coast-production-view-clusters" created
+```
+
+Download a kubeconfig for the east-coast-production service account from Tectonic Console. This kubeconfig will be submitted as a secret to the replica cluster to read from the directory cluster’s API.
+
+## Set up replica cluster(s)
+
+Switch the kubeconfig to the replica cluster. Check that it is set up correctly and can talk to the cluster:
+
+```
+export KUBECONFIG=/path/to/kubeconfig
+kubectl get nodes
+```
+
+Upload the kubeconfig that was just downloaded as a secret, for the syncer to use:
+
+```
+kubectl -n tectonic-system create secret generic \
+    tectonic-multi-cluster-local-sync --from-file=kubeconfig=/path/to/sa-east-coast-production-token-<id>-<clustername>-kubeconfig
```

Comment:
The name of the secret should be `directory-sync` instead of `tectonic-multi-cluster-local-sync`